Syntaxes/Swift.tmLanguage

```diff
@@ -4041,10 +4041,38 @@
 					<string>storage.type.function.swift</string>
 				</dict>
 				<dict>
+					<key>captures</key>
+					<dict>
+						<key>1</key>
+						<dict>
+							<key>name</key>
+							<string>keyword.other.accessiblity-modifier.swift</string>
+						</dict>
+						<key>2</key>
+						<dict>
+							<key>name</key>
+							<string>punctuation.definition.arguments.begin.swift</string>
+						</dict>
+						<key>3</key>
+						<dict>
+							<key>name</key>
+							<string>keyword.other.accessiblity-modifier.arguments.swift</string>
+						</dict>
+						<key>4</key>
+						<dict>
+							<key>name</key>
+							<string>punctuation.definition.arguments.end.swift</string>
+						</dict>
+						<key>5</key>
+						<dict>
+							<key>name</key>
+							<string>keyword.other.accessiblity-modifier.swift</string>
+						</dict>
+					</dict>
 					<key>match</key>
-					<string>(?&lt;!\.)\b(?:fileprivate|private|internal|public|open)\b</string>
+					<string>(?&lt;!\.)\b(fileprivate|private|internal|public|open)\s*(?:(\()(set)(\)))?</string>
 					<key>name</key>
-					<string>keyword.other.declaration-specifier.accessibility.swift</string>
+					<string>meta.keyword.accessibility-modifiers.swift</string>
```

Comment:
I don't think there's any precedent for `meta.keyword`. How about `meta.property` or `meta.directive`? (As far as I know there is no exact science to this, unless @infininight has a better suggestion)